kafka-ui-react-app/src/redux/reducers/connect/selectors.ts

```diff
@@ -52,6 +52,19 @@ export const getFailedConnectors = createSelector(
   }
 );
 
+export const getSortedTopics = createSelector(
+  connectState,
+  ({ connectors }) => {
+    return connectors.map((connector) => {
+      return connector.topics?.length
+        ? [...connector.topics].sort((a: string, b: string) =>
+            a.localeCompare(b)
+          )
+        : connector.topics;
+    });
```

Comment:
return connector.topics?.sort((a, b) => a.localeCompare(b))

Reply:
As far as I remember, localeCompare is a very slow operation and should only be used for strings containing non-latin characters.

Reply:
fixed

Reply:
```diff
- return connector.topics?.length
-         ? [...connector.topics].sort()
-         : connector.topics;
+ return connector.topics?.sort((a, b) => a.localeCompare(b))
```

Reply:
fixed